### DISCUSSION TITLE
Win+G hotkey

### DISCUSSION THREAD

**Discussion by tmcdaniel-aspyr** [valinet/ExplorerPatcher#2412] [Ideas]
Firstly, fantastic project!!! I wouldn't be using Windows 11 without it. Have to have my taskbar on the side.

This is really a plea for help, and unrelated to ExplorerPatcher, but since the project devs have obviously deep knowledge of Windows internals, maybe someone here might have a clue...

I've searched and searched for a way to prevent the Win+G hotkey from bringing up the Game Bar overlay. None of the recommended settings or reg hacks work. I know the Game Bar can be uninstalled, but when I do that, whenever I launch a game, you get a popup complaining about not having an app to handle ms-gameoverlay protocol, or something like that, and directing you to the Windows Store to find an app. This whole issue wouldn't be such a problem, but I use keyboard/mouse sharing software (Synergy) to share my PC KB and mouse to my Mac, and when pressing Win+G as a legitimate hotkey for Mac (== Cmd+G), the Game Bar pops up on the PC and eats the hotkey.

**Comment by pyrates999** [comment]
look at clavier+ @ http://utilfr42.free.fr/

**Reply by tmcdaniel-aspyr** [reply]
Thanks for the suggestion. Unfortunately, Clavier+ isn't able to disable or override Win+G.

**Reply by pyrates999** [reply]
Oh ok.  I use Clavier+ to disable right shift+enter by adding it as a new shortcut, but setting it to do nothing.  And in chrome, that prevents a new windows from then opening when you type in the address bar and accidentally hit right shift+enter.

**Reply by pyrates999** [reply]
After press win + g, I clicked the settings button in xbox game bar and set it to off.  Now win + g doesn't open xbox gamebar.

**Reply by tmcdaniel-aspyr** [reply]
You're probably on Win 10? On Win 11, there is no option to turn it off.

**Reply by pyrates999** [reply]
Here's where I turned it off in settings:

![image](https://github.com/valinet/ExplorerPatcher/assets/106135298/5ac038bf-9780-45ab-8729-e83fade4b8a0)

**Reply by tmcdaniel-aspyr** [reply]
In Windows 11, there is no option to turn it off completely, only the option to disable the controller button.
![image](https://github.com/valinet/ExplorerPatcher/assets/125918476/23483634-975e-42ea-a7e7-ce8dd20437b8)
